tests/common/config_reload.py

```diff
@@ -21,7 +21,18 @@ def config_system_checks_passed(duthost, delayed_services=[]):
         logging.info("Checking failure reason")
         fail_reason = duthost.shell("systemctl list-units --state=failed", module_ignore_errors=True)
         logging.info(fail_reason['stdout_lines'])
-        return False
+        # For kvm testbed
+        # Service watchdog-control loads failed because of the expected Error `No module named 'sonic_platform'.
+        # Service system-health loads failed because of the failure of the above service.
+        # So in kvm testbed, if these two services load fail, we consider the system is healthy.
```

Comment:
Probably the better idea is to raise an issue for the `watchdog-control` service not loaded on kvm testbed.

Reply:
I have raised an issue and use conditional mark to skip this script temporarily.